podman_compose.py

```diff
@@ -35,8 +41,6 @@
 import yaml
 from dotenv import dotenv_values
 
-__version__ = "1.0.4"
```

Comment:
I think it was fine if both setup.cfg and podman_compose.py duplicated the version. 

The dependency problems is a valid justification to do so. The downside is that these versions can get out of sync. However this can be addressed by having CI run a simple regex based test that extracts versions from files and compares them.